assets/templates/selector.tmpl

```diff
@@ -36,6 +36,9 @@
                                                 <span class="ons-radio ons-radio--no-border">
                                                     <input type="radio" id="{{ .Value }}" class="ons-radio__input ons-js-radio" value="{{ .Value }}" name="dimension" {{ if eq $.InitialSelection .Value }} checked {{ end }}>
                                                     <label class="ons-radio__label" for="{{ .Value }}" id="{{ .Value }}-label">{{ .Label }} ({{ thousandsSeparator .TotalCount }})</label>
+                                                    <div class="ons-radio__other ons-u-fs-s ons-u-pb-no">
+                                                        {{ .Description }}
+                                                    </div>
```

Comment:
This could benefit from being surrounded by an if, I happen to be reviewing a dataset that doesn't have the descriptions set (although they should be) and when there's no description the browser injects whitespace
```suggestion
                                                    {{ if .Description }}
                                                          <div class="ons-radio__other ons-u-fs-s ons-u-pb-no">
                                                              {{- .Description -}}
                                                          </div>
                                                    {{ end }}
```

Reply:
Updated code with this suggestion